legend-engine-testable/src/main/java/org/finos/legend/engine/testable/api/Testable.java

```diff
@@ -48,24 +53,27 @@
 
     private final TestableRunner testableRunner;
 
+    private static final ObjectMapper objectMapper = ObjectMapperFactory.getNewStandardObjectMapperWithPureProtocolExtensionSupports();
```

Comment:
Should we load this while initialization.
If we load in this fashion it would be susceptible to thread context this class gets loaded in.
What do you think?